LLIO2025/src/services/TaskApiService.ts

```
@@ -1,91 +1,120 @@
+      user_id: task.userId || 1,
+      project_id: task.projectId || 1,
+      category_id: task.categoryId || 1,
+    };
+    
+    try {
+        const response = await POST<typeof taskForApi, { reponse: string, task: any }>("/tasks", taskForApi);
+        
+        // Extraire l'objet task de la réponse
+        let taskData: any;
+        
+        if (response && response.task) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)